indico/modules/events/abstracts/controllers/management.py

```diff
@@ -16,16 +16,25 @@
 
 from __future__ import unicode_literals
 
-from flask import redirect, flash, jsonify
+from collections import defaultdict
+from os import path
```

Comment:
Don't import it like this; `import os` and then later use `os.path.join` - `path` is a way too common name to have it as a global.